nucleus/slice.py

```diff
@@ -27,3 +27,38 @@ def info(self, id_type: str = DATASET_ITEM_ID_KEY) -> dict:
         }
         """
         return self._client.slice_info(self.slice_id, id_type)
+
+    def append(
+        self, 
+        dataset_item_ids: List[str] = None,
+        reference_ids: List[str] = None
+    ) -> dict:
+        """
+        Appends to a slice from items already present in a dataset.
+        The caller must exclusively use either datasetItemIds or reference_ids
+        as a means of identifying items in the dataset.
+
+        :param
+        dataset_item_ids: List[str],
+        reference_ids: List[str],
+
+        :return:
+        {
+            "slice_id": str,
+        }
+        """
+        if dataset_item_ids and reference_ids:
+            raise Exception(
+                "You cannot both dataset_item_ids and reference_ids"
```

Comment:
grammar error with message here: "You cannot specify both dataset_item_ids and reference_ids"